test/load.test.js

```diff
@@ -127,4 +127,28 @@ test('dont load CSV and see if you get the correct response (4 INVALID_SPEEDs)',
     t.same(resp, [ 4294967295, 4294967295, 4294967295 ], "Verify expected speed results");
     t.end();
   });
-});
\ No newline at end of file
+});
+
+test('annotator without coordinates support - by default', function(t) {
+  const coordsFalse = new bindings.Annotator();
+  coordsFalse.loadOSMExtract(path.join(__dirname,'data/winthrop.osm'), (err) => {
+    if (err) throw err;
+    var coords = [[-120.1872774,48.4715898],[-120.1882910,48.4725110]];
+    coordsFalse.annotateRouteFromLonLats(coords, (err) => {
+      t.equals(err.toString(), 'Error: Annotator not created with coordinates support');
```

Comment:
Here and below - can we simply check for error if we check for the exact string then we have to keep it in sync in case the Node bindings slightly change the error message string.